scaffolds_actor_model/elixir/test/actor_model_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule ActorModelTest do
+  use ExUnit.Case
+  doctest ActorModel
+
+  test "greets the world" do
+    assert ActorModel.hello() == :world
```

Comment:
Shouldn't we add some tests for the worker themselves?

Reply:
Added, and also extracted the greeting logic in another module